hedy.py

```diff
@@ -1469,12 +1469,27 @@ def list_access(self, meta, args):
             return args[0] + '[' + args[1] + '-1]'
 
     def add(self, meta, args):
-        value = self.process_variable(args[0], meta.line)
+        # only process arg if arg is a string, else keep as is (ie. don't change 5 into '5', my_list[1] into 'my_list[1]')
+        if args[0].isnumeric(): # is int/float
+            value = args[0]
+        elif ('[' in args[0] and ']' in args[0]): # is list indexing
+            before_index, after_index = args[0].split(']', 1)
+            value = before_index + '-1' + ']' + after_index # account for 1-based indexing
+        else:
+            value = self.process_variable(args[0], meta.line) 
+
         list_var = args[1]
         return f"{list_var}.append({value})"
 
     def remove(self, meta, args):
-        value = self.process_variable(args[0], meta.line)
+        if args[0].isnumeric(): # is int/float
```

Comment:
Would be good to repeat the comment from add here (future readers will not always think to read add when looking for more info on remove)
```suggestion
      # only process arg if arg is a string, else keep as is (ie. don't change 5 into '5', my_list[1] into 'my_list[1]')
        if args[0].isnumeric(): # is int/float
```